core/src/main/java/nl/nn/adapterframework/util/AppConstants.java

```diff
@@ -71,7 +72,10 @@ private AppConstants(ClassLoader classLoader) {
 	 * @return AppConstants instance
 	 */
 	public static AppConstants getInstance() {
-		return getInstance(null);
```

Comment:
Misschien is het handiger deze changes uit de PR te halen. Dit wil ik los op gaan pakken en dan voorkomen we een merge conflict :)